apple/internal/bundle_package_type.bzl

```diff
@@ -0,0 +1,27 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Support for bundle package type used by Apple bundling rules."""
+
+# Bundle package type identifiers
+# These correspond to the CFBundlePackageType key in Info.plist files.
+bundle_package_type = struct(
```

Comment:
for completeness, could add `DEXT` for `com.apple.package-type.wrapper.driver-extension` and `SYSX` for `com.apple.package-type.wrapper.system-extension` and `""` for `com.apple.package-type.in-app-purchase-content`, but not necessary

Reply:
`rules_apple` doesn't build these yet. Figured I'd constrain it to just what it's aware of for now.

Reply:
yup totally makes sense, I just grepped through the xcspec and thought I'd share what else came up